<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>customParametersWidget</class>
 <widget class="QWidget" name="customParametersWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	background-color: #dfdfdf;
	font: 600 12pt &quot;Segoe UI&quot;;
	color: #324b4c;
}

QScrollBar:vertical {
        width: 20px;
        background: none;
    }

QScrollBar::add-line:vertical, QScrollBar::sub-line:vertical {
        background: #dfdfdf;
    }

QPushButton {
	background-color: #c9c9c9;
	border: none;
	border-radius: 10px;
	padding: 5px;
	font-size: 10pt;
}

QPushButton:hover{
	background-color: #aeaeae;
}

QDateEdit::up-button, QDateEdit::down-button{
	width: 0px;
}


QLineEdit, QDateEdit{
	border: none;
	font-size: 11pt;
	border-bottom: 1px solid #7c9399;
}

QMenu {
	border: 1px solid #aeaeae;
	border-radius: 5px;
}

QMenu::item {
	padding: 5px;
}

QMenu::item:selected {
	background-color: #aeaeae;
	padding-left: 7px;
}

yearsBtnsHolder QPushButton:clicked {
	background-color: black;
}

#optionsHolder, #optionsHolder_2 {
	border-bottom: 2px dashed #7c9399;
}

#pageTitle {
	font: 700 13pt &quot;Segoe UI&quot;;
}

#userIcon, #logoutBtn{
	background-color: #dfdfdf;
	padding: 0px;
}

#logoutBtn:hover{
	icon: url(:/icons/app/assets/icons/Log-Out--Streamline-Feather_#192626.svg);
}


#usernameLabel {
	font-size: 11pt;
}

#workerPlaceLineEdit, #workerPositionLineEdit {
	font-weight: 700;
}

#widget_3 * {
	font-size: 11pt;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>5</number>
   </property>
   <property name="rightMargin">
    <number>5</number>
   </property>
   <property name="bottomMargin">
    <number>5</number>
   </property>
   <item alignment="Qt::AlignTop">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="headerHolder" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>20</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item alignment="Qt::AlignLeft">
          <widget class="QWidget" name="widget_4" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <property name="spacing">
             <number>10</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
           </layout>
          </widget>
         </item>
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="pageTitle">
           <property name="text">
            <string>НОМЕНКЛАТУРИ</string>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignRight">
          <widget class="QWidget" name="userHolder" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <property name="spacing">
             <number>10</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>5</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QPushButton" name="logoutBtn">
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../../../resources.qrc">
                <normaloff>:/icons/app/assets/icons/Log-Out--Streamline-Feather.svg</normaloff>:/icons/app/assets/icons/Log-Out--Streamline-Feather.svg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="userIcon">
              <property name="focusPolicy">
               <enum>Qt::NoFocus</enum>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../../../resources.qrc">
                <normaloff>:/icons/app/assets/icons/User--Streamline-Feather.svg</normaloff>:/icons/app/assets/icons/User--Streamline-Feather.svg</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="usernameLabel">
              <property name="text">
               <string>admin</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item alignment="Qt::AlignLeft">
       <widget class="QWidget" name="parametersInfoHolder" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>35</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>5</number>
         </property>
         <item alignment="Qt::AlignLeft|Qt::AlignTop">
          <widget class="QWidget" name="parametersHolder" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <property name="spacing">
             <number>10</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QWidget" name="widget_7" native="true">
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <property name="spacing">
                <number>10</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QPushButton" name="cehoveBtn">
                 <property name="text">
                  <string>Работни места</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="machinesBtn">
                 <property name="text">
                  <string>Машини</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="workingPosBtn">
                 <property name="text">
                  <string>Длъжности</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="operTypesBtn">
                 <property name="text">
                  <string>Видове Длъжност</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="yarnsBtn">
                 <property name="text">
                  <string>Прежди</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="usersBtn">
                 <property name="text">
                  <string>Потребители</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item alignment="Qt::AlignLeft">
             <widget class="QPushButton" name="addNewEntryBtn">
              <property name="text">
               <string>Добавяне</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>888</width>
        <height>673</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="parametersTableHolder" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <property name="spacing">
           <number>5</number>
          </property>
          <property name="leftMargin">
           <number>5</number>
          </property>
          <property name="topMargin">
           <number>5</number>
          </property>
          <property name="rightMargin">
           <number>5</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTableView" name="parametersTableView">
            <property name="styleSheet">
             <string notr="true">QHeaderView:section{
	font: 700 10.5pt &quot;Segoe UI&quot;;
	background-color: #dfdfdf;
	padding: 5px;
}
QVerticalView:section{
	min-height: 30;
}
QAbstractItemView{
	alternate-background-color: #d3d3d3;
	font: 10.5pt &quot;Segoe UI&quot;;
	selection-background-color: rgba(198, 228, 254, 45);
	selection-color: #324b4c;
	focus: none;
}
QAbstractItemView:item{
	selection-background-color: rgba(198, 228, 254, 45);
	selection-color: #324b4c;
	focus: none;
}
QAbstractItemView::item:selected{
	background-color: rgba(198, 228, 254, 45);
}</string>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="cornerButtonEnabled">
             <bool>false</bool>
            </property>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../resources.qrc"/>
 </resources>
 <connections/>
</ui>
